WalletWasabi.Fluent/Behaviors/ShowAttachedFlyoutWhenFocusedBehavior.cs

```diff
@@ -56,6 +56,7 @@ protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
 		if (IsFlyoutOpen)
 		{
 			controller.SetIsForcedOpen(false);
+			IsFlyoutOpen = false;
```

Comment:
You can remove the line above because the `IsOpenPropertySynchronizer` will do that job.